VikingQuiz.UI/src/Components/StartGame/StartGame.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from 'react';
+import './StartGame.css';
+import HomeButton from '../Buttons/HomeButton/HomeButton';
+import axios from '../../../node_modules/axios';
+import UserMinimalProfile from '../UserMinimalProfile/UserMinimalProfile';
+import CancelButton from '../Buttons/CancelButton/CancelButton';
+import StartButton from '../Buttons/StartButton/StartButton';
+
+class StartGame extends React.Component<any, any> {
+    constructor(props: any) {
+      super(props);
+
+        this.state = {
+            serverMessage: '',
+            redirect: false,
+            player: [],
+            currentPlayerNumber: 0
+        }
+    }
+
+    public componentWillMount() {
+        axios.get('http:///localhost:60151/api/player')
+        .then(response => {
+            console.log(response.data)
+            this.setState({player: response.data})
+        })
+        .catch(err => console.log(err))
+    }
+
+    public getFirstListOfPlayers() {
+        const firstListOfPlayers = [];
+        for( let i = 0; i < 7; i++){
```

Comment:
This magic number 7 should be extracted in a constant in case in the future somebody would like it to become 8 or 10 or 5.